boot/freeldr/bootsect/ntfs.S

```diff
@@ -302,6 +301,12 @@ BootPartition:
 BootFlagSignature:
     .word HEX(0aa55)    // BootSector signature
 
+// End of Bootsector
+// Next sector starts as sector 2, since boot sector is 
+// as a file under NTFS ($Boot, which has inode number 7)
+// and takes the two first clusters of the volume (which means)
+// 8 sectors (checked on Windows and mkfs.ntfs from ntfsutils)
+
```

Comment:
IIRC cluster size can vary between different NTFS instances? Thus two clusters don't really means 8 sectors in all cases, right? Or only NTFS instances those have 1 cluster = 4 sectors can be used for boot? BTW 1 sector on PC HDD usually is 512 bytes in the old days, and a NTFS cluster by default is 4KiB, that means a NTFS cluster by default equals 8 sectors, not 4 sectors.

Reply:
For the moment, it only boots on sectors with 512 bytes, as it's the most common NTFS settings, see the TODO list in the pull request.
Also I messed up with my calculations, it's 16 sectors for two clusters, thanks for pointing it :)